homeassistant/components/eq3btsmart/config_flow.py

```diff
@@ -0,0 +1,131 @@
+"""Config flow for eQ-3 Bluetooth Smart thermostats."""
+
+import logging
+from typing import Any
+
+from homeassistant import config_entries
+from homeassistant.components.bluetooth import BluetoothServiceInfoBleak
+from homeassistant.const import CONF_MAC, CONF_NAME
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.device_registry import format_mac
+
+from .const import CONF_RSSI, DOMAIN
+from .schemas import SCHEMA_NAME, SCHEMA_NAME_MAC
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class EQ3ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow for eQ-3 Bluetooth Smart thermostats."""
+
+    def __init__(self) -> None:
+        """Initialize the config flow."""
+
+        self.name = ""
+        self.mac = ""
+        self.rssi = 0
```

Comment:
I'm not sure we should show the rssi here as it doesn't seem relevant unless its failing to setup